python/lsst/ts/m2gui/utility_monitor.py

```diff
@@ -881,3 +911,71 @@ def update_position(
                 position[idx] = round(component, self.NUM_DIGIT_AFTER_DECIMAL)
 
             signal.emit(position)
+
+    def update_net_force_moment_total(
+        self,
+        component_x: float,
+        component_y: float,
+        component_z: float,
```

Comment:
The component is in fact 3d vector. Probably easier (as you will spare building a vector) would be to pass:

vector: list[float]

and when you call that on messages, use list comprehension:

update_net_force_moment_total([message["f" + axis] for axis in "xyz"], ..)

Reply:
Fixed in the commit: [e5a46e3](https://github.com/lsst-ts/ts_m2gui/pull/45/commits/e5a46e3e907479c4ce0e51b07878165a73b2cad5) Thanks!